### ISSUE TITLE
Openpose has no effect on any picture.

### ISSUE THREAD

**Issue by ca3gamedev** [lllyasviel/ControlNet#157]
open pose doesn't work neither on automatic1111 nor comfyUI.

already used both the 700 pruned model and the kohya pruned model as well.

neither has any influence on my model.
Also I click enable and also added the anotation files.

Open pose simply doesnt work.

neither the open pose editor can generate a picture that works with the open pose control net.

Control picture just appears totally or totally black.

**Comment by dilectiogames** [comment]
update the webui extension again. did the same for me today, now after update it works again

**Comment by VenomSnake01** [comment]
silimar problem on my webUI. how to fix?

Canny work well.
Openpose, not working at all.

There is NO output annotator pic when i generate with openpose preprocessor..

**Comment by dilectiogames** [comment]
some of the controlnet modes will require additional downloads apart those 5Gb models you have downloaded manually. The first time you chose a model and you hit Generate it will download files that are up to 1Gb in size. Some tiems after an update it will download them again (or maybe some updated ones) Check the console if there is any error about not being able to access an url. This can happen if some antivirus blocks the connection

**Comment by dilectiogames** [comment]
never mind, I think is broken. This is what I get by always updating, lol.

TypeError: ControlParams.__init__() takes 9 positional arguments but 10 were given

go figure waht could be

**Comment by dilectiogames** [comment]
third time the charm. enabled and disabled the extension a couple of times, restarting the Ui, restarting.  the entire thing. now it works. crisis adverted

**Comment by itsRoov** [comment]
im experiencing the same problem, updated webui to the latest, updated controlnet (which promptly broke my webui and made it become stuck on 'installing requirements', but regardless...) and openpose ends up having 0 effect on img gens

**Comment by eugeneslin** [comment]
Set preprocessor to none. Seems weird but it works.

**Comment by dilectiogames** [comment]
Set preprocessor to none works when you don't need to calculate the input. For example if you already have the depth image you want to use you can skip the creation of this image by using a preprocessor.

**Comment by naoki23** [comment]
Hi. I experienced same issue. I also set preprocessor to none and it works fine.
Openpose preprocessor seemed to be loaded , but additional downloads did not start.
```
Loading model: control_openpose-fp16 [9ca67cc5]
Loaded state_dict from [/notebooks/stable-diffusion-webui/extensions/sd-webui-controlnet/models/control_openpose-fp16.safetensors]
ControlNet model control_openpose-fp16 [9ca67cc5] loaded.
Loading preprocessor: openpose
  0%|                                                    | 0/20 [00:00<?, ?it/s]
  5%|██▏                                         | 1/20 [00:01<00:20,  1.10s/it]
 10%|████▍                                       | 2/20 [00:01<00:14,  1.22it/s]
 15%|██████▌                                     | 3/20 [00:02<00:12,  1.37it/s]
 20%|████████▊                                   | 4/20 [00:02<00:11,  1.45it/s]
```

I am using Stable Diffusion Web UI on Paperspace Gradient, and the Web UI was set up using a Paperspace template.
Web UI and sd-webui-controlnet have been updated to the latest version.

Is it possible to manually download additional download data?

**Comment by baston1** [comment]
Try using with another browser, in my case I spent time trying to get it to work on bravo, but it didn't work. I used chrome and openpose and others worked.

**Comment by rinme** [comment]
> Try using with another browser, in my case I spent time trying to get it to work on bravo, but it didn't work. I used chrome and openpose and others worked.

Yeah. This is worked for me, I switched to chrome and it's working.

**Comment by ha-yi** [comment]
The pre-processor is acting as annotator, used to prepare the raw images. If you already have a pose image (RGB colored stick) then its already been annotated. you just need to set it to none if you are using it. But if you are using a real photograph, or some sketch drawing, use the preprocessor to annotate the input image, for example: manga or sketch images can be annotated using canny or scribble.

**Comment by WillyamBradberry** [comment]
Just changed model from 
control_v11p_sd15_openpose.pth 
to 
control_sd15_openpose [fef5e48e]

Work for me.
